main/station-config.js

```diff
@@ -2,6 +2,7 @@
 
 const Store = require('electron-store')
 const { randomUUID } = require('crypto')
+const { resolve } = require('path')
```

Comment:
what is this for?

Reply:
It's a draft. I'll let you know when it's ready for review